app/controllers/admin/custom_value_sets_controller.rb

```diff
@@ -2,21 +2,21 @@ class Admin::CustomValueSetsController < Admin::AdminController
   def create
     @record = CustomValueSet.new(record_params)
     authorize @record
-    @record.save
+    @record.save!
```

Comment:
There being no error handling here, I thought it better to raise an exception rather than return a success message.